Crash due to lazy `exoPlayer` property initialization

- [x] I have verified there are no duplicate active or recent bugs, questions, or requests

###### Include the following:
 - ExoMedia version: `5.0.0` (pulled from Jitpack, https://github.com/brianwernick/ExoMedia/commit/743fa9757cc9d898218467dc08c252a7c6c22e8d)
 - Device OS version: `11.0`
 - Devide Manufacturer: `Google`
 - Device Name: `Pixel 3`
 
###### Reproduction Steps

Set up basic `VideoView` in a project,  use it with the following code:

```java
videoView = findViewById(R.id.videoView);
videoView.setVideoURI(Uri.parse(url));
videoView.setOnPreparedListener(() -> videoView.start());
```

###### Expected Result

No crash, playback starts

###### Actual Result

Crash with stacktrace

```
     Caused by: java.lang.NullPointerException
        at com.google.android.exoplayer2.util.Assertions.checkNotNull(Assertions.java:156)
        at com.google.android.exoplayer2.analytics.AnalyticsCollector.generateEventTime(AnalyticsCollector.java:851)
        at com.google.android.exoplayer2.analytics.AnalyticsCollector.generateCurrentPlayerMediaPeriodEventTime(AnalyticsCollector.java:802)
        at com.google.android.exoplayer2.analytics.AnalyticsCollector.notifySeekStarted(AnalyticsCollector.java:181)
        at com.devbrackets.android.exomedia.nmp.ExoMediaPlayerImpl.seekTo(ExoMediaPlayerImpl.kt:338)
        at com.devbrackets.android.exomedia.nmp.ExoMediaPlayerImpl.seekTo(ExoMediaPlayerImpl.kt:334)
        at com.devbrackets.android.exomedia.core.video.ExoVideoPlayer.setMedia(ExoVideoPlayer.kt:116)
        at com.devbrackets.android.exomedia.core.audio.AudioPlayerApi$DefaultImpls.setMedia$default(AudioPlayerApi.kt:117)
        at com.devbrackets.android.exomedia.ui.widget.VideoView.setVideoURI(VideoView.kt:351)
```

###### Workaround

Looked into why this happens, turns out in `ExoMediaPlayerImpl` the `exoPlayer` property isn't initialized by any of the actions performed above, and therefore the call made to the analytics collector [here](https://github.com/brianwernick/ExoMedia/blob/743fa9757cc9d898218467dc08c252a7c6c22e8d/library/src/main/kotlin/com/devbrackets/android/exomedia/nmp/ExoMediaPlayerImpl.kt#L338) crashes internally, because its `player` property gets set when the `exoPlayer` property is initialized and executes the builder, connecting the analytics collector to it.

Using this as a workaround currently, which triggers the creation of the `exoPlayer` property internally:

```java
videoView = findViewById(R.id.videoView);
videoView.isPlaying(); // Workaround to init some internals of the library
videoView.setVideoURI(Uri.parse(url));
videoView.setOnPreparedListener(() -> videoView.start());
```

--- a/library/src/main/kotlin/com/devbrackets/android/exomedia/listener/OnPreparedListener.kt
+++ b/library/src/main/kotlin/com/devbrackets/android/exomedia/listener/OnPreparedListener.kt
@@ -4,7 +4,7 @@ package com.devbrackets.android.exomedia.listener
  * Interface definition for a callback to be invoked when the media
  * source is ready for playback.
  */
-interface OnPreparedListener {
+fun interface OnPreparedListener {
   /**
    * Called when the media file is ready for playback.
    */

--- a/library/src/main/kotlin/com/devbrackets/android/exomedia/nmp/ExoMediaPlayerImpl.kt
+++ b/library/src/main/kotlin/com/devbrackets/android/exomedia/nmp/ExoMediaPlayerImpl.kt
@@ -42,20 +42,18 @@ class ExoMediaPlayerImpl(
   private val listeners = CopyOnWriteArrayList<ExoPlayerListener>()
   private val rendererListener = DelegatedRenderListener()
 
-  private val exoPlayer: ExoPlayer by lazy {
-    ExoPlayer.Builder(
-      config.context,
-      config.rendererFactory,
-      config.mediaSourceFactory,
-      config.trackManager.selector,
-      config.loadControl,
-      config.bandwidthMeter,
-      config.analyticsCollector
-    ).build().also {
-      it.addListener(this)
-      it.addListener(rendererListener)
-      it.addListener(config.analyticsCollector)
-    }
+  private val exoPlayer: ExoPlayer = ExoPlayer.Builder(
+    config.context,
+    config.rendererFactory,
+    config.mediaSourceFactory,
+    config.trackManager.selector,
+    config.loadControl,
+    config.bandwidthMeter,
+    config.analyticsCollector
+  ).build().also {
+    it.addListener(this)
+    it.addListener(rendererListener)
+    it.addListener(config.analyticsCollector)
   }
 
   private val stopped = AtomicBoolean()

